script.sh

```diff
@@ -0,0 +1,4 @@
+!#/bin/bash
+find /home/vvi/part1 -name "*.[ch]*" -exec cp {} /tmp/guesanumber/  \; || echo "no files"
+tar cfz /tmp/guesanumber/arc.tgz /tmp/guesanumber/*
+mv /tmp/guesanumber/arc.tgz /home/vvi/part1/release
```

Comment:
hard-coding /home/vvi/part1/release is bad idea. Please use relative path instead.